controller/scheduler/swarm.py

```diff
@@ -48,14 +52,20 @@ def stop(self, name):
         """
         Stop a container
         """
-        self.docker_cli.stop(name)
+        try:
+            self.docker_cli.stop(name)
+        except:
+            raise RuntimeError("cannot stop container " + name)
 
     def destroy(self, name):
         """
         Destroy a container
         """
-        self.stop(name)
-        self.docker_cli.remove_container(name)
+        try:
+            self.stop(name)
+            self.docker_cli.remove_container(name)
+        except:
```

Comment:
See above. References:
- https://www.python.org/dev/peps/pep-0020/ "The Bible" lines 10-11
- http://c2.com/cgi/wiki?CatchWhatYouCanHandle